driver-core/src/test/functional/com/mongodb/internal/connection/AwsAuthenticationSpecification.groovy

```diff
@@ -31,7 +31,7 @@ class AwsAuthenticationSpecification extends Specification {
 
         when:
         openConnection(connection, async)
-        executeCommand(getConnectionString().getDatabase(), new BsonDocument('count', new BsonString('test')), connection)
+        executeCommand(getConnectionString().getDatabase(), new BsonDocument('count', new BsonString('test')), null, connection)
```

Comment:
The API of executeCommand changed with versioned API support, but it was missed here because the test was being skipped at the time and also because Groovy.  Fixed by adding a null ServerAPI instance to the calls.

Reply:
Ack.